e2e_tests/tests/task/test_generic_tasks.py

```diff
@@ -168,6 +176,13 @@ def test_generic_task_create_with_fork() -> None:
     expected_config = {"entrypoint": ["echo", "forked"]}
     assert result_config == expected_config
 
+    task.wait_for_task_state(
+        test_session, task_resp.taskId, bindings.v1GenericTaskState.COMPLETED, timeout=30
```

Comment:
I prefer if ```timeout=30``` is just a default value for the parameter in the function

Then we can override it if there are specific cases where we should only wait a time period. Most of these seem like a "just wait long enough"